src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Service/Customizations/Skillsets/SkillConstructors/EntityRecognitionSkill_BackCompatCtror.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Please use the same file naming convention that we use elsewhere (suffix with `.Customization.cs`)